juju-destroy-model-example.sh

```diff
@@ -18,6 +18,9 @@ set -ux
 : ${CONTROLLER_NAME:="${OS_PROJECT_NAME}-${CLOUD_NAME}"}
 : ${MODEL_NAME:="${OS_PROJECT_NAME:0:12}"}
 
-juju switch ${CONTROLLER_NAME}:${MODEL_NAME} &&\
-    juju destroy-model --destroy-storage $1 ${CONTROLLER_NAME}:${MODEL_NAME} ||\
-        echo "Controller:Model not found (${CONTROLLER_NAME}:${MODEL_NAME})"
+juju switch ${CONTROLLER_NAME}:${MODEL_NAME} || \
+    echo "Controller:Model not found (${CONTROLLER_NAME}:${MODEL_NAME})"
```

Comment:
This message was previously printed if destroy-model failed, not if the switch failed.  I see that it is difficult to retain the message on the same place due to the use of timeout builtin.

But, does it make sense to continue with destroy and kill if the switch fails?  I guess they will just fail too, so bug by bug compatible at least :)

Reply:
True, if the model is missing it would have thrown a nasty error, and missed getting to model not found ;-)